### PR TITLE
base of motions implemented

### PR DESCRIPTION
**Pull request by evidentti**

### CODE REVIEW

File: app/scripts/controllers/meeting.details.ctrl.js [hunk 1]
```diff
@@ -292,12 +297,22 @@ angular.module('dashboard')
             $rootScope.$emit(CONST.UNSAVEMEETINGDDATA, (self.hasUnsavedProposal || self.remarkIsUnsaved));
         });
 
-        $scope.$on('$destroy', unsavedRemarkWatcher);
-        $scope.$on('$destroy', unsavedProposalWatcher);
-        $scope.$on('$destroy', proposalCountWatcher);
+        var motionsWatcher = $rootScope.$on(CONST.MOTIONSUPDATED, function (aEvent, aData) {
+                $log.debug("meetingDetailsCtrl.motionsWatcher: ", arguments);
+                if (angular.isObject(aData)) {
```
Comment (tkenakka) [inline app/scripts/controllers/meeting.details.ctrl.js:302]: https://docs.angularjs.org/api/ng/type/$rootScope.Scope: "you can compare the newVal and oldVal. If these two values are identical (===) then the listener was called due to initialization."
Okay maybe the value difference check is better to be left out like you did?

File: app/scripts/motions.directive.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            };
+
+            self.typeString = function (id) {
+                var result = null;
+                angular.forEach(CONST.MOTIONTYPES, function (value) {
+                    if (angular.isObject(value) && value.id === id) {
+                        result = value.stringId;
+                    }
+                }, this);
+                return result ? result : '-';
```
Comment (tkenakka) [inline app/scripts/motions.directive.js]: No need for the check if `result`is initialised to the default `'-'` ?

File: app/views/motions.directive.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            </div>
+            <div class="list-group-item db-padding-10-hor" ng-repeat="m in c.motions | orderBy : 'publishDate' track by $index" ng-click="c.selectMotion(m)">
+                <div class="row">
+                    <span class="col-xs-7 col-md-5 db-flex">
+                    <span class="db-margin-10-right center glyphicon ng-class:{'glyphicon-minus':c.isSelected(m), 'glyphicon-plus':!c.isSelected(m)}" aria-hidden="true"></span>
+                    <b class="db-flex-area db-truncate">{{m.title}}</b>
+                    </span>
+                    <b><i class="col-xs-5 col-md-3 db-truncate">{{m.personName}}</i></b>
+                    <span class="col-md-2 hidden-xs hidden-sm db-truncate" translate translate-cloak>{{c.typeString(m.type)}}</span>
+                    <span ng-show="m.supporters" class="col-md-2 hidden-xs hidden-sm db-truncate">{{m.supporters.length}} <span ng-if="c.supporting(m.supporters)" class="glyphicon glyphicon-user db-margin-10-hor"></span></span>
```
Comment (tkenakka) [inline app/views/motions.directive.html]: Please add an "signed by me" tooltip for the icon

Comment (tkenakka) [inline app/views/motions.directive.html]: The "Signed by me" info is missing in xs and sm resolution.

File: app/views/motions.directive.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+                </div>
+                <div class="panel-collapse collapse" uib-collapse="!c.isSelected(m)">
+                    <div class="hidden-md hidden-lg db-margin-05-top">
+                        <span translate translate-cloak>STR_SIGNING_TYPE</span>:<span class="db-margin-05-left" translate
+                            translate-cloak>{{c.typeString(m.type)}}</span>
+                    </div>
+                    <div ng-if="m.supporters.length">
+                        <span translate translate-cloak>STR_SROLE_SUPPORTER</span>:<span ng-repeat="s in m.supporters track by $index"><span ng-if="0 < $index">, </span>
```
Comment (tkenakka) [inline app/views/motions.directive.html]: Please change the label text from "supporter" to "signeeS"

**Comment by tkenakka** [comment]
As discussed, the btn count should be fetched at state contstruction and the location for the btn should probably be moved to meeting.status.html.

**Comment by tkenakka** [comment]
LGTM with comments

File: app/scripts/controllers/meeting.details.ctrl.js (hunk 1 above)
Comment (evidentti) [reply app/scripts/controllers/meeting.details.ctrl.js:302]: Value is a number. I do not see benefits to check it.

**Pull request merged**